lib/matplotlib/__init__.py

```diff
@@ -760,7 +858,8 @@ def gen_candidates():
 
 _deprecated_ignore_map = {}
 
-_obsolete_set = {'text.dvipnghack', 'legend.isaxes'}
+_obsolete_set = {'text.dvipnghack', 'legend.isaxes', 'verbose.level',
+                 'verbose.fileo'}
```

Comment:
Why did you deprecated them?  I the log level should stay configurable from the mplrc?